dist/stat_data.py

```diff
@@ -244,6 +244,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_pages_queued_urgent', 'pages queued for urgent eviction'),
     CacheStat('cache_eviction_pages_seen', 'pages seen by eviction walk'),
     CacheStat('cache_eviction_pages_already_queued', 'pages seen by eviction walk that are already queued'),
+    CacheStat('cache_eviction_inparallel_with_checkpoint', 'eviction and checkpoint operations running in parallel'),
```

Comment:
`inparallel` -> `in_parallel`?

Reply:
Also can you change the description to indicate that this stat is counted for each page eviction?

Reply:
Changed and also included `pages` in the stat along with the description.